code/datums/traits/good.dm

```diff
@@ -62,7 +62,7 @@
 /datum/quirk/fan_clown
 	name = "Clown Fan"
 	desc = "You enjoy clown antics and get a mood boost from wearing your clown pin."
-	value = 1
+	value = 4
```

Comment:
```suggestion
	value = 2
```

Reply:
This would bring it in line with the values you set for Mime Fan (its counterpart) and Musician.